hasura.planx.uk/migrations/1702312905340_alter_view_analytics_summary_add_analytics_ended_at/up.sql

```diff
@@ -0,0 +1,22 @@
+CREATE OR REPLACE VIEW "public"."analytics_summary" AS 
+ SELECT a.id AS analytics_id,
+    al.id AS analytics_log_id,
+    f.slug AS service_slug,
+    t.slug AS team_slug,
+    a.type AS analytics_type,
+    a.created_at AS analytics_created_at,
+    a.user_agent,
+    a.referrer,
+    al.flow_direction,
+    al.metadata,
+    al.user_exit AS is_user_exit,
+    al.node_type,
+    al.node_title,
+    al.has_clicked_help,
+    al.input_errors,
+    (date_part('epoch'::text, (al.next_log_created_at - al.created_at)))::numeric(10,1) AS time_spent_on_node_seconds,
+    a.ended_at AS analytics_ended_at
```

Comment:
_Comment:_

I couldn't decide whether to do this or directly calculate the analytics session length in minutes like `time_spent_on_node_seconds`

Although I though `analytics_ended_at` could be more versatile and the calculation could be done in the GUI 🤷

Reply:
Focusing on versatility makes sense to me! Does it have to be one or the other though, could it simply be both - the raw timestamp column _and_ a calculation column? Just asking, happy for this to merge either way :+1: